src/main/java/com/rakovets/course/java/core/practice/oop_classes_and_objects/Time.java

```diff
@@ -0,0 +1,61 @@
+package com.rakovets.course.java.core.practice.oop_classes_and_objects;
+
+public class Time {
+    private int hours;
+    private int minutes;
+    private int seconds;
+
+    {
+        this.hours = 0;
+        this.minutes = 0;
+        this.seconds = 0;
+    }
+
+    Time() {
+    }
+
+    Time(int totalSeconds) {
+        final int secondsInMin = 60;
+        final int secondsInHour = 3600;
+        final int secondsInDay = 86400;
```

Comment:
"Violation: 'NAME_CONSTANT' was not used for constant naming
Solution: Rename constant"